components/ArticlesList.jsx

```diff
@@ -0,0 +1,44 @@
+import ArticleCard from "./ArticleCard";
+import { useState, useEffect } from "react";
+
+function ArticlesList() {
+
+    const [articlesArray, setArticlesArray] = useState([]);
```

Comment:
we also need an isLoading state here to let the user know the webpage is working the articles are just still loading